src/rmarkdown/preview.ts

```diff
@@ -147,22 +202,39 @@ export class RMarkdownPreviewManager {
     // (e.g., is an unopened tab)
     public async showSource(): Promise<void> {
         if (this.activePreview) {
-            await vscode.commands.executeCommand('vscode.open', this.activePreview.uri, {
+            await vscode.commands.executeCommand('vscode.open', this.activePreview?.uri, {
                 preserveFocus: false,
                 preview: false,
-                viewColumn: this.activePreview.resourceViewColumn ?? this.activePreview.panel.viewColumn ?? vscode.ViewColumn.Active
+                viewColumn: this.activePreview?.preview?.resourceViewColumn ?? this.activePreview?.preview?.panel.viewColumn ?? vscode.ViewColumn.Active
             });
         }
     }
 
     public openExternalBrowser(): void {
         if (this.activePreview) {
             void vscode.env.openExternal(
```

Comment:
Opening a local html file end up as a new tab in vscode here. Not sure if there is an easy way to open it in default browser.

Reply:
https://github.com/sindresorhus/open seems to get around this issue, and works as expected when replacing openExternal. I guess we could either go with that, or just remove openExternal

Reply:
We could definitely use it.

Reply:
Holding off using it just for the moment - I haven't found a good way of getting the default browser for file opening. It works if we hardcode it, but obviously not ideal to do so

Reply:
I revert to using `vscode.env.openExternal` at the moment. To make it work on both local and remote, user might need [Live Preview](https://marketplace.visualstudio.com/items?itemName=ms-vscode.live-server) to serve the HTML. Let's keep it simeple first.